src/app/seok/kmong/components/category.tsx

```diff
@@ -0,0 +1,158 @@
+import { ChevronDown } from 'lucide-react'
+import Link from 'next/link'
+import { useState } from 'react'
+
+export const Category = () => {
+  const [isOpen, setIsOpen] = useState(false)
+  const categories1 = [
+    '디자인',
+    '마케팅',
+    '영상·사진·음향',
+    'IT·프로그래밍',
+    '전자책·템플릿',
+  ]
+  const designDetails = {
+    로고브랜딩: ['로고 디자인', '브랜드 디자인·가이드'],
+    인쇄홍보물: [
+      '명함',
+      '전단지·포스터·인쇄물',
+      '현수막·X배너',
+      '메뉴판',
+      '홍보물 인쇄·출력',
+      '스티커·봉투·초대장',
+    ],
+    웹모바일디자인: [
+      '웹 UI·UX',
+      '앱·모바일 UI·UX',
+      '템플릿형 홈페이지',
+      '아이콘·버튼',
+    ],
+    마케팅디자인: [
+      '상세페이지',
+      'SNS·썸네일 디자인',
+      '채널아트 디자인',
+      '방송용 아바타',
+      '배너·배달어플',
+      '블로그·카페 디자인',
+    ],
+    패키지커버: ['책표지·내지', '전자책 표지·내지', '앨범커버', '패키지'],
+    캐릭터일러스트: [
+      '일러스트',
+      '캐리커쳐',
+      '웹툰·콘티',
+      '2D 캐릭터',
+      '이모티콘',
+    ],
+  }
+  const categories2 = [
+    '상위 2% Prime',
+    '포트폴리오',
+    '프리랜서클럽',
+    '크몽로그',
+  ]
+  const businessCategories = [
+    '디자인',
+    'IT·프로그래밍',
+    '영상·사진·음향',
+    '마케팅',
+    '번역·통역',
+    '문서·글쓰기',
+    '창업·사업',
+    '주문제작',
+    '세무·법무·노무',
+  ]
+
+  const careerCategories = ['취업·입시', '투잡·노하우', '직무역량 레슨']
+
+  const hobbyCategories = ['운세', '심리상담', '취미 레슨', '생활서비스']
+  return (
+    <div className="">
+      <div className="w-[1200px] flex items-center mx-auto pl-[14px]">
+        <div
+          className="flex group cursor-pointer relative"
+          onMouseEnter={() => setIsOpen(true)}
+          onMouseLeave={() => setIsOpen(false)}>
+          <div className="flex py-4">
+            <span className="font-extrabold">전체 카테고리</span>
```

Comment:
폰트가 너무 두꺼워요ㅠ_ㅠ bold로도 충분할 듯
```suggestion
<span className="font-bold">전체 카테고리</span>
```